src/go-binaries-all-versions/script.py

```diff
@@ -0,0 +1,59 @@
+# Copyright (C) 2023 Mandiant, Inc. All Rights Reserved.
+'''
+This Python script automates the process of generating Go binaries for versions 1.11 to 1.20 using Docker. By leveraging Docker containers, it ensures a consistent and reproducible environment for compiling the Go source code across different platforms.
+
+Please note that Docker must be installed and properly configured on your system for the script to execute successfully.
+'''
+
+import subprocess
+import sys
+
+go_versions = """
+go1.11
+go1.12
+go1.13
+go1.14
+go1.15
+go1.16
+go1.17
+go1.18
+go1.19
+go1.20
+"""
+
+subprocess.call(["docker", "compose", "up"])
+print("Compose is UP")
+
+for version in go_versions.split():
+    print("version: ", version[2:])
```

Comment:
if we only use `version[2:]` can we just store the version numbers without `go`?